src/promptflow/promptflow/_sdk/_submitter/test_submitter.py

```diff
@@ -412,31 +415,73 @@ def flow_test(
+                    )
+                    flow_outputs = {}
+                    node_run_infos = None
+                    aggregation_inputs = None
+                    run_info = None
+                    for chunk in line_result:
+                        node_run_infos = chunk.node_run_infos if node_run_infos is None else node_run_infos
+                        aggregation_inputs = chunk.aggregation_inputs if aggregation_inputs is None else aggregation_inputs
+                        run_info = chunk.run_info if run_info is None else run_info
+                        for key, value in chunk.output.items():
+                            if key not in flow_outputs:
+                                flow_outputs[key] = value
+                            else:
+                                flow_outputs[key] += value
+
+                    flow_outputs = persist_multimedia_data(
+                        flow_outputs, base_dir=self.flow.code, sub_dir=Path(".promptflow/output")
                     )
-                    line_result.node_run_infos.update(aggregation_results.node_run_infos)
-                    line_result.run_info.metrics = aggregation_results.metrics
-                if isinstance(line_result.output, dict):
-                    # Remove line_number from output
-                    line_result.output.pop(LINE_NUMBER_KEY, None)
-                    generator_outputs = self._get_generator_outputs(line_result.output)
-                    if generator_outputs:
-                        logger.info(f"Some streaming outputs in the result, {generator_outputs.keys()}")
-                return line_result
+                    if aggregation_inputs:
+                        # Convert inputs of aggregation to list type
+                        flow_inputs = {k: [v] for k, v in inputs.items()}
+                        aggregation_inputs = {k: [v] for k, v in aggregation_inputs.items()}
+                        aggregation_results = async_run_allowing_running_loop(
+                            flow_executor.exec_aggregation_async, flow_inputs, aggregation_inputs
+                        )
+                        node_run_infos.update(aggregation_results.node_run_infos)
+                        run_info.metrics = aggregation_results.metrics
+                    if isinstance(flow_outputs, dict):
+                        # Remove line_number from output
+                        flow_outputs.pop(LINE_NUMBER_KEY, None)
+                        logger.info(f"Some streaming outputs in the result, {flow_outputs.keys()}")
+                    flow_result = LineResult(output=flow_outputs, aggregation_inputs=aggregation_inputs, run_info=run_info, node_run_infos=node_run_infos)
+                    self._raise_error_when_test_failed(flow_result, show_trace=True)
+                    show_node_log_and_output(flow_result.node_run_infos, kwargs.pop("show_step_output", False), generator_record)
```

Comment:
server return node output/result is a dict and can;t be read by python
![image](https://github.com/microsoft/promptflow/assets/26239730/0ceeb780-2775-478b-81a5-670ba60ec030)

Reply:
not show node output/result